notebooks/ch-algorithms/grover.ipynb

```diff
@@ -1286,8 +1286,8 @@
     "result = sv_sim.run(grover_circuit).result()\n",
     "statevec = result.get_statevector()\n",
     "statevec\n",
-    "#from qiskit_textbook.tools import vector2latex\n",
-    "#vector2latex(statevec, pretext=\"|\\\\psi\\\\rangle =\")"
+    "from qiskit_textbook.tools import vector2latex\n",
```

Comment:
(just  a preference thing but) should this `import` statement be moved to the top of the cell/code block?

Reply:
No strong preference from me; I think it's slightly easier to read the cell this way, i.e. "simulate, import fn, use fn" vs "import fn, simulate, use fn", but it's maybe more common to keep imports at the top.

Reply:
no problem. it can remain as is!